### ISSUE TITLE
POP namelist variables that are controlled by XML

### ISSUE THREAD

**Issue by mnlevy1981** [marbl-ecosys/MARBL#130]
There are several variables in POP that the user should not be able to change via `user_nl_pop`, and many of them are ecosystem-related. @klindsay28 can you help expand this list?

Low-hanging fruit:
```
 abio_dic_dic14_on
 cfc_on
 ciso_on
 ecosys_on
 iage_on
 irf_on
 sf6_on
```
are all set according to what's in `OCN_TRACER_MODULES`

**Comment by mnlevy1981** [comment]
(assigning @mvertens because this will work into her pythonization of `build-namelist`)

**Comment by klindsay28** [comment]
Here are some (probably more to come later):
chl_option depends upon OCN_CHL_TYPE
lactive_ice depends upon OCN_ICE_FORCING
atm_co2_opt depends upon OCN_CO2_TYPE
lvariable_PtoC will depend upon OCN_TRACER_MODULES_OPT

**Comment by mnlevy1981** [comment]
Closed via [ESCOMP/POP2-CESM#4](https://github.com/ESCOMP/POP2-CESM/issues/4)
